lib/functions/plot_measures.R

```diff
@@ -94,12 +131,62 @@ plot_measures <- function(
     plot_tmp <- plot_tmp +
       facet_wrap(vars({{ facet_var }}), ncol = 2)
   }
+  # Add y_scale to add option for free_y
+  if (!is.null(y_scale) && y_scale == "free_y") {
+    plot_tmp <- plot_tmp +
+      facet_wrap(~source, scales = "free_y")
+  }
+
+  # Add save_path option
+  if (!is.null(save_path)) {
+    ggsave(
+      filename = here("released_output", "results", "figures", save_path),
+      plot = plot_tmp,
+      width = 10,
+      height = 6
+    )
+  }
```

Comment:
I think we should have this as a separate function call after each plot, otherwise this function is doing too many thing?